runtime/oti/VMHelpers.hpp

```diff
@@ -2080,13 +2134,13 @@ class VM_VMHelpers
 	 * @return true if we need to scan the java stack
 	 */
 	static VMINLINE bool
-	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject, bool scanOnlyUnmounted = false)
+	needScanStacksForContinuation(J9VMThread *vmThread, j9object_t continuationObject)
 	{
 		bool needScan = false;
 #if JAVA_SPEC_VERSION >= 19
 		jboolean started = J9VMJDKINTERNALVMCONTINUATION_STARTED(vmThread, continuationObject);
 		J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, continuationObject);
-		needScan = started && (NULL != continuation) && (!scanOnlyUnmounted || !isContinuationMounted(continuation));
+		needScan = started && (NULL != continuation) && (!isContinuationMountedOrConcurrentlyScanned(continuation));
```

Comment:
let's an add a comment 
We don't scan mounted continuation:
- for concurrent GCs, since stack is actively changed. Instead, we scan them during preMount or during root scanning if already mounted at cycle start or during postUnmount (might be indirectly via card cleaning) or during final STW (via root re-scan) if still mounted at cycle end
- for sliding compacts to avoid double slot fixups
For fully STW GCs, there is no harm to scan them, but it's waste of time since they are scanned during root scanning already.
 We don't scan currently scanned either - one scan is enough.